src/components/ScrollWheel/index.tsx

```diff
@@ -11,27 +12,27 @@ enum WHEEL_QUADRANT {
   RIGHT = 4,
 }
 
-enum KEY_CODE {
-  ARROW_UP = 38,
-  ARROW_DOWN = 40,
-  ARROW_LEFT = 37,
-  ARROW_RIGHT = 39,
-  ESC = 27,
-  ENTER = 13,
-  SPACE = 32,
-}
-
-const centerClickEvent = new Event('centerclick');
-const centerLongClickEvent = new Event('centerlongclick');
-const forwardScrollEvent = new Event('forwardscroll');
-const backwardScrollEvent = new Event('backwardscroll');
-const wheelClickEvent = new Event('wheelclick');
-const menuClickEvent = new Event('menuclick');
-const menuLongPressEvent = new Event('menulongpress');
-const backClickEvent = new Event('backclick');
-const forwardClickEvent = new Event('forwardclick');
-const playPauseClickEvent = new Event('playpauseclick');
-const idleEvent = new Event('idle');
+type SupportedKeyCode =
```

Comment:
Updating the logic for keyboard events to not use key codes anymore, since they are deprecated: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode